### PR TITLE
Add Lithuanian

### PR DESCRIPTION
**Pull request by TheOnlyGhostwolf**
hewwo? Hewwo!?! Pwease, awwrowe my puww rewquest (　･ω･)

### CODE REVIEW

File: strings.json [hunk 1]
```diff
@@ -5,4 +5,11 @@
     "members": "{{count}} Membros",
     "button": "Juntar-se"
   }
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
  },
```

File: strings.json [hunk 2]
```diff
@@ -5,4 +5,11 @@
...
+  "lt-LT": {
+    "header": "Tu buvai pakviesta(s) prisijungti prie serverio",
+    "online": "{{count}} prisijungę",
+    "members": "{{count}} nariai",
+    "button": "Prisijungti"
+  }
+
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
```

File: strings.json [hunk 3]
```diff
@@ -5,4 +5,11 @@
     "members": "{{count}} Membros",
     "button": "Juntar-se"
   }
+  "lt-LT": {
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
  "lt": {
```

**Comment by TheOnlyGhostwolf** [comment]
Fixed

File: strings.json [hunk 4]
```diff
@@ -4,5 +4,11 @@
     "online": "{{count}} online",
     "members": "{{count}} Membros",
     "button": "Juntar-se"
+  },
+  "lt": {
+    "header": "Tu buvai pakviesta(s) prisijungti prie serverio",
+    "online": "{{count}} prisijungę",
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
    "online": "{{count}} prisijungęs",
```

File: strings.json [hunk 5]
```diff
@@ -4,5 +4,11 @@
...
+    "members": "{{count}} nariai",
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
    "members": "{{count}} Nariai",
```

File: strings.json [hunk 6]
```diff
@@ -4,5 +4,11 @@
     "online": "{{count}} online",
     "members": "{{count}} Membros",
     "button": "Juntar-se"
+  },
+  "lt": {
+    "header": "Tu buvai pakviesta(s) prisijungti prie serverio",
```
Comment (pedrofracassi) [inline strings.json]:
```suggestion
    "header": "Tu buvai pakviestas prisijungti prie serverio",
```

I know, I know. But they should match Discord's strings.

**Comment by TheOnlyGhostwolf** [comment]
Should be good for now

**Pull request merged**